dotcom-rendering/cdk/bin/cdk.ts

```diff
@@ -111,7 +111,7 @@ new RenderingCDKStack(cdkApp, 'InteractiveRendering-PROD', {
 	stage: 'PROD',
 	domainName: 'interactive-rendering.guardianapis.com',
 	scaling: {
-		minimumInstances: 6,
+		minimumInstances: 3,
 		maximumInstances: 24,
```

Comment:
I don't mind allowing it to scale up to 24 - it feels like a generous safely threshold in case something does go wrong and it needs to scale up a couple of times? It shouldn't normally go above 3 (or 6 during deploys) otherwise